petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -45,3 +45,15 @@ def test_primitive(self):
         assert (ts.short_col.dtype.type == np.int16)
         assert (ts.int_col.dtype.type == np.int32)
         assert (ts.long_col.dtype.type == np.int64)
+
```

Comment:
Add a test, checking the tf.data.Dataset can read all the rows from the spark dataframe.